cluster-sync/sync.sh

```diff
@@ -180,10 +180,15 @@ if [ "${CDI_SYNC}" == "test-infra" ]; then
   _kubectl apply -f "./_out/manifests/test-proxy.yaml"
   _kubectl apply -f "./_out/manifests/sample-populator.yaml"
   _kubectl apply -f "./_out/manifests/uploadproxy-nodeport.yaml"
-  # Imageio test service:
-  _kubectl apply -f "./_out/manifests/imageio.yaml"
-  # vCenter (VDDK) test service:
-  _kubectl apply -f "./_out/manifests/vcenter.yaml"
+
+  # Check if CPU architecture is s390x
+  if [ "$(uname -m)" != "s390x" ]; then
```

Comment:
So this would work if the host running the development env setup script is on s390x, does that cut it?
What if the developer is on x86 deploying custom CDI to a remote cluster? Maybe there's a better env var to use here for the s390x target cluster check?